### DISCUSSION TITLE
Sobre começar a palestrar

### DISCUSSION THREAD

**Discussion by marcusvcribeiro** [frontendbr/forum#847] [Discussão]
Galera, procurei alguns artigos de como começar a palestrar na Areá de TI, não achei nada do tipo.
Ai vim pedir aqui para a Galera que já palestra, contar como que eles organizam suas palestras, contar como foi as primeiras palestras para as pessoas que tem essa vontade, ter um rumo para preparar sua palestras.

**Comment by gabrieldarezzo** [comment]
Já ouviu?
https://devnaestrada.com.br/2017/02/17/a-arte-de-palestrar.html

**Comment by felipefialho** [comment]
Acho esse texto do @zenorocha bem legal para quem tem interesse em começar a palestrar:

- [O que eu aprendi depois de dar 100 palestras](https://tableless.com.br/o-que-eu-aprendi-depois-de-dar-100-palestras/)

**Comment by luisfmsouza** [comment]
Depois de fazer a minha primeira palestra eu escrevi esse texto explicando como foi o processo:

https://medium.com/@luisfmsouza/a-primeira-vez-a-gente-nunca-esquece-6279a36c75bc

Inclusive, cito um post do Nando Vieira sobre o mesmo assunto.

PS1: esse post já tem mais de 1 ano
PS2: não sou uma exímio palestrante

**Comment by matheusml** [comment]
Oi @marcusvcribeiro, excelente pergunta! 

#### Why
Primeiramente você deve [começar com o porquê](https://www.amazon.com.br/Start-Why-Leaders-Inspire-Everyone/dp/1591846447/ref=sr_1_1?ie=UTF8&qid=1512749904&sr=8-1&keywords=start+with+why). Você quer divulgar conteúdo? Ficar famoso na comunidade? Viajar o país? Seja lá qual for a resposta, é interessante você saber o porquê de querer palestrar.
Palestrar é bem difícil, então se você não souber o que te motiva, é bem provável que você desista na primeira dificuldade que aparecer no seu caminho.


#### How
Agora que você já sabe o porquê, falta saber o como.
Na minha opinião, o mais fácil é começar de baixo. Vá pra meetups locais, converse bastante com o pessoal, preste bastante atenção em como os outros palestrantes se comportam e o que eles fazem. Depois de um certo tempo, peça pra algum dos organizadores do meetup pra fazer uma palestra rápida. De 5 a 10min é o suficiente. Faça isso de novo e de novo, até se sentir confortável.

Depois disso, suba um pouco o nível. Comece a mandar CFP pra eventos pequenos. Depois eventos grandes. Fazendo isso aos poucos, passo-a-passo, vai chegar uma hora que você não vai mais precisar ir atrás dos eventos/meetups, eles é que vão atrás de você.

O importante é ter humildade e começar pequeno. Esqueça a BrazilJS, ou o Front in Sampa, por enquanto pelo menos.

Espero ter ajudado. Abraços!

**Comment by liviagabos** [comment]
Existem muitos eventos de faculdades, como semanas de informática, que você pode se disponibilizar a falar de um assunto do seu interesse.

Eu começo com a ideia que mensagem eu quero passar. Qual o objetivo da palestra em uma frase?
Baseado nisso, eu preciso saber quem é o meu público. Isso é importante para você saber de onde você vai começar a falar e no que você precisa focar.
Exemplo: Eu quero falar de requisitos de acessibilidade de interface web para desenvolvedores backend web. Ok, então eles entendem da parte web e eu não preciso falar de acessibilidade em vídeos, só em formulários.
Assim, você vai delimitando para saber o que você precisa explicar, o que você pode assumir que eles já sabem e o que você só citar para eles pesquisarem mais depois.

Sempre deixando links de rodapé ou referências no final do que você acha importante.

**Comment by woliveiras** [comment]
Fala, pessoal!

Foi criado um projeto pela galera do @training-center para ajudar pessoas que desejam começar a palestrar.

É uma ideia de ajudar a pessoa a montar a palestra, revisar tudo e a pessoa ainda mandar um vídeo dela apresentando. Caso a pessoa se sinta confortável, poderá fazer a palestra via hangouts para o pessoal do Training Center ou via live também para que outras pessoas também lhes dê feedbacks.

https://github.com/training-center/help4papers
